javascript/operators/object_initializer.json

```
@@ -127,7 +127,7 @@
                 "version_added": false
               },
               "nodejs": {
-                "version_added": true
+                "version_added": "6.0.0"
```

This might be from 4.0.0? https://node.green/#ES2015-syntax-object-literal-extensions-shorthand-methods

```suggestion
                "version_added": "4.0.0"
```